src/Interface/Modules/Render/ES/RendererCollaborators.h

```diff
@@ -258,11 +258,11 @@ namespace SCIRun
       explicit ObjectScaleCalculator(const BasicRendererObjectProvider* s, const ScaleParameters& p);
       gen::Transform computeTransform(int x, int y) const override;
     private:
-      glm::vec3 originView_;
+      glm::vec3 originView_ {0.0};
       float projectedW_;
-      glm::vec3 flipAxisWorld_;
-      glm::vec3 originToSpos_;
-      glm::vec3 originWorld_;
```

Comment:
With so many of these needing new zero-initialization, perhaps derive a type that default-constructs with zeros? `struct SCIVec3` or something

Reply:
I agree, but that name sounds like the native Vector was duplicated. How about GLMVec3?